src/hooks/useKyberSwapConfig.ts

```diff
@@ -66,7 +66,7 @@ const parseGlobalResponse = (
 ): KyberswapGlobalConfig => {
   const data = responseData?.data?.config
   const aggregatorDomain = data?.aggregator ?? AGGREGATOR_API
-  const isEnableAuthenAggregator = !!data?.isEnableAuthenAggregator
+  const isEnableAuthenAggregator = !data ? true : !!data?.isEnableAuthenAggregator
```

Comment:
What's this for?

Reply:
for anonymous user?

Reply:
this config decide use access token or not.
update to true in default

Reply:
yep. or when fetching

Reply:
should we create a const that `DEFAULT_is_Enable_Authen_Aggregator = true`?